platform/mellanox/docker-syncd-mlnx-rpc.mk

```diff
@@ -20,6 +20,8 @@ SONIC_INSTALL_DOCKER_IMAGES += $(DOCKER_SYNCD_MLNX_RPC)
 endif
 
 $(DOCKER_SYNCD_MLNX_RPC)_CONTAINER_NAME = syncd
+$(DOCKER_SYNCD_MLNX_RPC)_VERSION = 1.0.0
+$(DOCKER_SYNCD_MLNX_RPC)_PACKAGE_NAME = syncd
```

Comment:
syncd rpc is used for debugging only. For SONiC applications there is no difference wether it is syncd or syncd rpc since both provide same redis db interface.

Reply:
I need to generated a suffix for version when package is debug, 1.0.0-dbg

Reply:
Done